src/mock_crust/crust.rs

```diff
@@ -73,13 +73,13 @@ impl Service {
     /// Start service discovery (beacon).
     /// Note: beacon is not yet implemented in mock.
     pub fn start_service_discovery(&mut self) {
-        trace!("[MOCK] start_service_discovery not implemented in mock");
+        trace!(target: "mock_crust", "[MOCK] start_service_discovery not implemented in mock");
```

Comment:
This should probably be `crust`. (Or the readme above should list `mock_crust`.)